_posts/2022-1-19-quantization-in-practice.md

```diff
@@ -0,0 +1,292 @@
+---
+layout: blog_detail
+title: 'Quantization in Practice'
+author: Suraj Subramanian, Jerry Zhang
+featured-img: ''
+---
+
+There are a few different ways to quantize your model with PyTorch. In this blog post, we'll take a look at how each technique looks like in practice. I will use a non-standard model that is not traceable, to paint an accurate picture of how much effort is really needed when quantizing your model.
+
+<div class="text-center">
+  <img src="/assets/images/quantization_gif.gif" width="60%">
+</div>
+
+## A quick introduction to quantization
```

Comment:
would you like to integrate some part of this in the official quantization api?

Reply:
do you mean the API itself, or in the docs? I'm happy to add it!

Reply:
the documentation, we can add the explanations of core things like qconfig, quantized tensor, observer/fake_quant, qscheme to documentation: https://pytorch.org/docs/master/quantization.html#